contracts/Splitter.sol

```diff
@@ -50,8 +50,8 @@ contract Splitter is Pausable {
 
         balances[msg.sender]  = SafeMath.sub(msgSender, amount);
         emit WithDraw(msg.sender, amount);
-        msg.sender.transfer(amount);
-
+        (bool success, ) = msg.sender.call{value: amount}("");
+        require(success, "Transfer failed");
         return true;
```

Comment:
Since you know that `success == true`, then you might as well return that value. Or rather declare it `returns (bool success)` and set it `(success, ) = msg.sender...`.